Packs/ApiModules/Scripts/HTTPFeedApiModule/HTTPFeedApiModule.py

```diff
@@ -94,13 +97,26 @@ def __init__(self, url: str, feed_name: str = 'http', insecure: bool = False, cr
             self.polling_timeout = int(polling_timeout)
         except (ValueError, TypeError):
             raise ValueError('Please provide an integer value for "Request Timeout"')
+
         self.headers = headers
         self.encoding = encoding
         self.feed_name = feed_name
         if not credentials:
             credentials = {}
-        self.username = credentials.get('identifier', None)
-        self.password = credentials.get('password', None)
+        self.username = None
```

Comment:
no need to init with `None`, lets just get it, same for password

Reply:
In case we have an API key in these fields, we need to have those as `None`.

Reply:
not sure i understand why, you just assign it to `header_value`, why do you need it to be `None` ?

Reply:
so I don't send nonsense in the http request, either username/password or API key header